packages/blockchain-wallet-v4-frontend/src/plugin/wallet-options-v4.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
@VitaliiShpital why is wallet-options-v4.json required here?

Reply:
before initial render of the application we call `configureStore` function and the very first step of it is fetching `wallet-options-v4.json` file

`// immediately load app configuration
  let options
  try {
    let res = await fetch('/wallet-options-v4.json')
    options = await res.json()
  } catch (e) {
    throw new Error('errorWalletOptionsApi')
  }`
  
  so for app to work we need this file to exist.
  
 As I mentioned above I decided to just copy this file from config/mocks into the built `lib` folder not to duplicate things

Reply:
Sorry I commented twice :)

Reply:
no worries :)